mod/state-transition/pkg/core/state_processor.go

```
@@ -200,6 +200,8 @@ func (sp *StateProcessor[
 		return nil, err
 	}
 
+	fmt.Println("val0", *validatorUpdates[0], "val1", *validatorUpdates[1])
```

golangci-lint (1.62.2)

use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)